modules/virt-creating-interface-on-nodes.adoc

```diff
@@ -41,12 +41,20 @@ spec:
               enabled: false
           port:
             - name: eth1
+    dns-resolver: <6>
```

Comment:
I don't think this will work. We need auto-dns: false in the bridge ipv4 section so there's a place to store the DNS settings.

Reply:
Fixed.